cmd/cmd.go

```diff
@@ -66,6 +68,19 @@ func newCmd() *cmd {
 			_, _ = fmt.Fprintln(ctx.App.Writer, command.Name)
 		}
 	}
+
+	debugFlags := &cli.BoolFlag{
+		Name:  "debug",
```

Comment:
only worked when this flag before any subcommand,

related: https://github.com/version-fox/vfox/issues/88